src/Krank/Formatter.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Krank.Formatter (
+  showViolations
+  ) where
+
+import Data.Text (Text, append, pack)
+
+import Krank.Types
+
+showViolations :: [Violation]
+               -> Text
+showViolations = foldl (\acc v -> acc `append` showViolation v) ""
```

Comment:
nit: `foldMap` is doing exactly that.

`foldMap showViolation`

Reply:
And one new tool in my toolbelt! thanks.

Reply:
Note that it works because text is a `Semigroup` (also a `Monoid`).